apps/web/pages/auth/setup/index.tsx

```diff
@@ -62,9 +119,21 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     };
   }
 
+  // Check existant CALCOM_LICENSE_KEY env var and acccount for it
+  if (process.env.CALCOM_LICENSE_KEY !== "" && !deployment?.licenseKey) {
+    deployment = await prisma.deployment.create({
+      data: {
+        id: 1,
+        licenseKey: process.env.CALCOM_LICENSE_KEY,
+        agreedLicenseAt: new Date(),
+      },
+    });
+  }
+
   return {
     props: {
       userCount,
+      deployment,
```

Comment:
We should never expose the licenseKey to the frontend. This will.

Reply:
Fixed, thx!